taskt/Core/Automation/Commands/Loop Commands/BeginMultiLoopCommand.cs

```diff
@@ -164,15 +164,15 @@ private void LoopConditionHelper_CellContentClick(object sender, DataGridViewCel
 
                     var automationCommands = taskt.UI.CustomControls.CommandControls.GenerateCommandsandControls().Where(f => f.Command is BeginLoopCommand).ToList();
                     frmCommandEditor editor = new frmCommandEditor(automationCommands, null);
-                    editor.selectedCommand = loopCommand;
-                    editor.editingCommand = loopCommand;
-                    editor.originalCommand = loopCommand;
-                    editor.creationMode = frmCommandEditor.CreationMode.Edit;
-                    editor.scriptVariables = ScriptVariables;
+                    editor.SelectedCommand = loopCommand;
+                    editor.EditingCommand = loopCommand;
+                    editor.OriginalCommand = loopCommand;
+                    editor.CreationModeInstance = frmCommandEditor.CreationMode.Edit;
+                    editor.ScriptVariables = ScriptVariables;
 
                     if (editor.ShowDialog() == DialogResult.OK)
                     {
-                        var editedCommand = editor.editingCommand as BeginLoopCommand;
+                        var editedCommand = editor.EditingCommand as BeginLoopCommand;
                         var displayText = editedCommand.GetDisplayValue();
                         var serializedData = Newtonsoft.Json.JsonConvert.SerializeObject(editedCommand);
```

Comment:
Could likely be reduced to JsonConvert.SerializeObject

Reply:
Same is the case here that it's just updating variable names/references in this Command File.